### DISCUSSION TITLE
Can't move emacs to workspace 2 0

### DISCUSSION THREAD

**Discussion by LinArcX** [WayfireWM/wayfire#1548] [Q&A]
Here is what I've done so far:

```
# Rules ────────────────────────────────────────────────────────────────────────
[window-rules]

emcas_maximize = on created if app_id is "Emacs" then maximize
emacs_move_to_workspce_20 = on created if app_id is "Emacs" then assign_workspace 2 0
```

But it's not working

Also when I run:
`WAYLAND_DEBUG=1 emacs 2>&1 | kak`
 
To get the properties of emacs, there is nothing in kak!

**Comment by ammen99** [comment]
Is emacs a wayland-native program? Perhaps it is running under Xwayland, and then you can find its WM_CLASS (for example with xprop) which is the equivalent of app-id for Xwayland clients.

**Reply by LinArcX** [reply]
> Is emacs a wayland-native program?

Don't know.

xprop output:
`WM_CLASS(STRING) = "emacs", "Emacs"`

Still not working with both `emacs` or `Emacs`.

**Comment by LinArcX** [comment]
It is worth mentioning that it's not working for Zathura, Speedcrunch applications either.

Zathura class = `org.pwmt.zathura`
Speedcrunch class = `org.speedcrunch.speedcrunch`
